### PR TITLE
Add PickPoint provider.

### PR DESCRIPTION
**Pull request by cylon-v**
This PR adds PickPoint geocoder, which is commercial version of Nominatim hosted at https://pickpoint.io.

### CODE REVIEW

File: src/Provider/PickPoint/PickPoint.php [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+use Geocoder\Query\ReverseQuery;
+use Geocoder\Http\Provider\AbstractHttpProvider;
+use Geocoder\Provider\LocaleAwareGeocoder;
+use Geocoder\Provider\Provider;
+use Http\Client\HttpClient;
+
+/**
+ * @author Vladimir Kalinkin <vova.kalinkin@gmail.com>
+ */
+final class PickPoint extends AbstractHttpProvider implements LocaleAwareGeocoder, Provider
```
Comment (Nyholm) [inline src/Provider/PickPoint/PickPoint.php]: Remove LocaleAwareGeocoder. See #720

File: src/Provider/PickPoint/Tests/IntegrationTest.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+use Geocoder\Provider\PickPoint\PickPoint;
+use Http\Client\HttpClient;
+
+/**
+ * @author Vladimir Kalinkin <vova.kalinkin@gmail.com>
+ */
+class IntegrationTest extends ProviderIntegrationTest
+{
+    protected $testAddress = true;
+    protected $testReverse = true;
```
Comment (Nyholm) [inline src/Provider/PickPoint/Tests/IntegrationTest.php]: This should be the other way around. Look in `ProviderIntegrationTest`. All tests are enabled by default, you should write ie `$testIpv4 = false`

Comment (cylon-v) [inline src/Provider/PickPoint/Tests/IntegrationTest.php]: @Nyholm, thank you for the review, I will fix it!

**Comment by cylon-v** [comment]
@Nyholm, requested changes are applied.

**Comment by Nyholm** [comment]
Great work

**Comment by Nyholm** [comment]
Oh, i forgot, you should update the table on the readme

**Pull request merged**